src/main/java/org/javacord/bot/commands/GradleCommand.java

```diff
@@ -21,7 +21,7 @@ public void onCommand(TextChannel channel) {
                           "  mavenCentral()\n" +
                           "}\n" +
                           "dependencies { \n" +
-                          "  implementation 'org.javacord:javacord:3.0.0'\n" + // TODO Always use the latest version
+                                  "  implementation 'org.javacord:javacord:3.0.1'\n" + // TODO Always use the latest version
```

Comment:
While not using the latest Javacord version from some request, at least use `Javacord.VERSION` in the meantime? But keep the TODO or better make an issue instead or additionally.